SPECS/xsconsole.spec

```diff
@@ -54,6 +54,10 @@ Console tool for configuring a XCP-ng installation.
 %{_unitdir}/xsconsole.service
 
 %changelog
+* Tue Aug 30 2022 Yann Dirson yann.dirson@vates.fr> - 10.1.13-1.2
```

Comment:
This is not exactly the date for today, and `<` is missing at the beginning of the e-mail address.

Reply:
I miss the emacs mode for .spec edition ;)